doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -372,7 +372,9 @@ Binary libraries
 Bluetooth libraries and services
 --------------------------------
 
-|no_changes_yet_note|
+* :ref:`bt_enocean_readme`:
+
+  * Fixed an issue where sensor data of a certain length would incorrectly be parsed as switch commissioning.
```

Comment:
```suggestion
  * Fixed an issue where the sensor data of a certain length was incorrectly parsed as switch commissioning.
```

Reply:
Btw, do we need to be more specific about at what length(s) this issue would be observed, instead of saying 'certain length'?

Reply:
I'm not sure, but I can add it.
```suggestion
  * Fixed an issue where the sensor data of length 18 bytes was incorrectly parsed as switch commissioning.
```
?

Reply:
of 18 bytes length :)